config/routes.rb

```diff
@@ -4,7 +4,12 @@
     get 'dashboard', to: 'dashboards#index', as: 'dashboard'
     get 'reports', to: 'reports#index', as: 'reports'
     post 'search', to: 'dashboards#search', defaults: { format: :js }
-    resources :users, only: [:new, :create, :index]
+
+    post 'users/search', to: 'users#search', as: 'users_search', defaults: { format: :js }
+    get 'users/:user_id/reset_password', to: 'users#reset_password', as: 'reset_password'
```

Comment:
in `get 'users/:user_id/reset_password`, change the `:user_id` to just `:id`. That follows the rails / rest pattern a bit closer, and will let us take advantage of the `find_user` before_action in the controller. Sound good?